components/lib/crash/src/main/java/mozilla/components/lib/crash/service/SentryService.kt

```diff
@@ -35,6 +37,7 @@ class SentryService(
             .build()
             .toString(),
         clientFactory).apply {
+        environment = env
```

Comment:
nitpicking level over 9000: If this is named `environment` in `SentryClient` maybe we should name it that too for consistency.